doc/10-icinga-template-library.md

```diff
@@ -3304,7 +3304,8 @@ fail2ban_path           | **Optional.** Specify the path to the tw_cli binary, d
 fail2ban_warning        | **Optional.** Specify a warning threshold, default is 1
 fail2ban_critical       | **Optional.** Specify a critical threshold, default is 2
 fail2ban_socket         | **Optional.** Specify a socket path, default is unset
-fail2ban_perfdata       | **Optional.** If set to true, activate the perfdata output, default value for the plugin is set to true.
+fail2ban_perfdata       | **Optional.** If set to true, activate the perfdata output, default value for the plugin is set to true
+fail2ban_jail           | **Optional.** Specify the name of the specific jail to monitor; default is to omit and thus monitor all jails.
```

Comment:
Please make this description the same as the one in the config file.

Reply:
There doesn't seem to be one? The -j parameter is not listed when you run check_fail2ban --help; it is however described in the documentation. See also: https://github.com/fail2ban/fail2ban/blob/master/files/nagios/check_fail2ban

Reply:
Ah sorry, I misunderstood; fix coming right up

Reply:
No, I mean you just adjusted the text (as I suggested) in the config file, but not in the docs.

Reply:
Fixed; sorry for the misunderstanding, sometimes I try to do things too fast! :-)